test/testFunctions.js

```diff
@@ -230,13 +294,14 @@ const checkUserReceivesTulipTokens = async (ST, BT, user) => {
   // userTulips = lockedTulFunds
   const newBalance = (await dx.balances.call(ST.address, user)).toNumber()
   console.log(`
-    USER'S OWNED TUL AMT = ${tulFunds}
-    USER'S LOCKED TUL AMT = ${lockedTulFunds}
+    USER'S OWNED TUL AMT  = ${tulFunds.toEth()}
+    USER'S LOCKED TUL AMT = ${lockedTulFunds.toEth()}
 
-    USER'S ETH AMT = ${newBalance}
+    USER'S ETH AMT = ${newBalance.toEth()}
   `)
-  // due to passage of time(stamp)
-  assert.isAtLeast(lockedTulFunds, tulips, 'final tulip tokens are slightly > than calculated from dx.claimBuyerFunds.call')
+  // with changes, TULIPS are NOT minted until auctionCleared
+  // lockedTulFunds should = 0
+  assert.isAtLeast(lockedTulFunds, 0, 'final tulip tokens are slightly > than calculated from dx.claimBuyerFunds.call')
```

Comment:
Could you please check here the exact amount?assert.isAtLeast->assert.equals.  I think it is needed to be sure everything worked